### ISSUE TITLE
Make ./clean_proto_deprecation.sh cross platform

### ISSUE THREAD

**Issue by kc1212** [dedis/cothority#1547]
The script that tries to tidy up java warnings in generated files doesn't work on macOS

```
./clean_proto_deprecation.sh
Fixing PARSER deprecation by using the public getter
sed: 1: "java/src/main/java/ch/e ...": invalid command code j
sed: 1: "java/src/main/java/ch/e ...": invalid command code j
sed: 1: "java/src/main/java/ch/e ...": invalid command code j
sed: 1: "java/src/main/java/ch/e ...": invalid command code j
sed: 1: "java/src/main/java/ch/e ...": invalid command code j
sed: 1: "java/src/main/java/ch/e ...": invalid command code j
sed: 1: "java/src/main/java/ch/e ...": invalid command code j
sed: 1: "java/src/main/java/ch/e ...": invalid command code j
sed: 1: "java/src/main/java/ch/e ...": invalid command code j
sed: 1: "java/src/main/java/ch/e ...": invalid command code j
sed: 1: "java/src/main/java/ch/e ...": invalid command code j
sed: 1: "java/src/main/java/ch/e ...": invalid command code j
sed: 1: "java/src/main/java/ch/e ...": invalid command code j
make[1]: *** [do-java] Error 1
make: *** [proto] Error 2
```

I think it's because the `sed` on macOS doesn't support `-i` the same way as the `sed` on GNU/Linux, see https://stackoverflow.com/questions/19456518/invalid-command-code-despite-escaping-periods-using-sed

**Comment by jeffallen** [comment]
This will be fixed in the authprox branch. (I had to fix it while rebasing into master.)

**Comment by jeffallen** [comment]
Fixed in 45c4cd913abf08489146322b09e244c206a0c2ba.
